src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -183,6 +209,17 @@ export class HuiCardOptions extends LitElement {
     fireEvent(this, "ll-edit-card", { path: this.path! });
   }
 
+  private _cutCard(): void {
+    this._copyCard();
+    fireEvent(this, "ll-cut-card", { path: this.path! });
```

Comment:
```suggestion
this._deleteCard();
```

Reply:
This brings the warning on cut `Are you sure you want to delete this card?`. 

I don't think I want that?

Reply:
Merged them into the same event, but with a confirm boolean. Seems a little cleaner.